runtime/src/chpl-visual-debug.c

```diff
@@ -51,6 +51,11 @@ static void cb_task_end(const chpl_task_cb_info_t *info);
 
 int chpl_vdebug_fd = -1;
 int chpl_vdebug = 0;
+int chpl_vdebug_task = 0;
```

Comment:
That may be possible, but I don't think it will be a problem. Here is my reasoning:

1) chpl_vdebug_task will be called once on each locale for each high level xxxxVdebug() call.
2) chpl_vdebug is not turned on and off as it was before, so we won't lose any events due to this being turned off. Only a pauseVdebug() or stopVdebug() will turn it off.
3) chpl_vdebug_task is used to only NOT generate messages, so if we have a race, we'll generate a record for file.
4) chpl_vdebug_task data is attached to all xxxxVdebug data so chplvis gets that data.
5) chpl_vdebug_task is set in the xxxxVdebug() calls before the coforall that will generate forks to other locales
6) chplvis filters out events associated with the xxxxVdebug calls that do generate records that can't be stopped in the runtime, and this should include items generated due to any race, if there.

This does bring up the question of do we want to limit where the xxxxVdebug() calls are allowed or do we want to lock at the top level of the xxxxVdebug() calls. I haven't explored what happens if one of the xxxxVdebug() calls are in an "on statement". This is where the real race may appear if concurrent tasks to both call an xxxxVdebug() calls.